src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -125,4 +125,24 @@ public String getNewQuestion(Entity game_entity, DatastoreService datastore) {
         return (update_game_entity.getProperty("quizQuestion")).toString();
     }
 
+    public Boolean giveUserPoints(Boolean userQuizStatus, Entity currentUser, DatastoreService datastore) {
+        if(currentUser == null || datastore == null) {
+            log.log(Level.SEVERE, "Given a null value in parameters");
```

Comment:
You don't need to do this if this method works, but it might be easier to do log.severe("Given a null value in parameters")

Reply:
You might also want to break this into two if statements so you can be more specific with the error messages

Reply:
Okay I broke them up!